includes/widgets/icon.php

```diff
@@ -304,13 +305,33 @@ protected function register_controls() {
 						'max' => 300,
 					],
 				],
-				'selectors' => [
-					'{{WRAPPER}} .elementor-icon' => 'font-size: {{SIZE}}{{UNIT}};',
+				'selectors'  => [
+					'{{WRAPPER}} .elementor-icon'     => 'font-size: {{SIZE}}{{UNIT}};',
```

Comment:
Fix spacing throughout the file - according to our standard, we only use single spaces between property names and operators. It's not linted currently, not sure why.